From: Free Ekanayaka <freee@debian.org>
Date: Thu, 5 Jan 2017 23:42:16 +0000
Subject: fix-flaky-service-hung-test

Increase the timeout used to detect if service process is
hung.
---
 tests/test_service.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_service.py b/tests/test_service.py
index 24cbcdd..97a871c 100644
--- a/tests/test_service.py
+++ b/tests/test_service.py
@@ -70,7 +70,7 @@ class ServiceIntegrationTest(TestCase):
         If the given command doesn't terminate with SIGTERM, it's SIGKILL'ed.
         """
         self.script.hang()
-        self.fixture.protocol.timeout = 0.2
+        self.fixture.protocol.timeout = 1.0
         self.fixture.expectOutput("hanging")
         self.fixture.setUp()
         self.fixture.cleanUp()
